### PR TITLE
Fix Small Key and Gilded Texture, Re-Add CSMC

### PR DESCRIPTION
**Pull request by Jaybone25**
PR is to make Small Key and Gilded Chest textures more visible/distinguishable when peeked from a distance or the side.   

Also added the option for old CSMC now called Classic at least temporarily until CTMC is more thoroughly tested and fine tuned.

Small Key Before:
Small Key Old

Small Key After
image

Major Item Before:
image

Major Item After:
image

### CODE REVIEW

**Comment by RawZ06** [comment]
Can you share a screenshot of the new chest textures with before/after ? Thank a lot.

**Comment by Jaybone25** [comment]
> Can you share a screenshot of the new chest textures with before/after ? For some viewers, we use a link to PR to show what ever is the setting to have screenshot instead of share discord oot dev-talk. Thank a lot.

Yup this is done!

**Comment by RawZ06** [comment]
Thank you :)

File: SettingsList.py [hunk 1]
```diff
@@ -3739,20 +3739,29 @@ def __init__(self, name, gui_text, min, max, default, step=1,
...
             'textures': 'Texture',
-            'sizes': 'Size and Texture'
+            'both':  'Both Size and Texture',
+            'classic': 'Classic'
         },
         gui_tooltip    = '''\
-            If enabled, chest texture will reflect its contents
+            
+			
+			If "Texture" is enabled, chest texture will reflect its contents
```
Comment (cjohnson57) [inline SettingsList.py:3748]: Sorry for the nitpick, but could you change these tabs to spaces? Everything else seems good to me!

Comment (Jaybone25) [reply SettingsList.py:3748]: Coulda sworn I got all those :(.   Hate how it adds tabs whenever you hit return lol, will fix tonight.

Comment (Cuphat) [reply SettingsList.py]: You might need to look into a better editor that can figure out tabs vs. spaces based on the file/project lmao.

> Sorry for the nitpick, but could you change these tabs to spaces? Everything else seems good to me!

I'd say not even really a nitpick, especially in a language that utilizes significant whitespace. :)

**Review by cjohnson57 — APPROVED** [review]

**Pull request merged**